### ISSUE TITLE
AI difficulty shows up as Elite when not elite

### ISSUE THREAD

**Issue by dsjoerg** [ggtracker/ggtrackerstack#10]
_From @eric-hu on August 13, 2015 21:19_

I thought I noticed this last month, was just chatting with a friend and he ran into this issue too.  Will post a link once I hear back from him.

_Copied from original issue: dsjoerg/esdb#3_

**Comment by dsjoerg** [comment]
Yes, this is a side-effect of all AI players being mapped to the same ESDB identity.  In order to fix it, I think the easiest way is to establish multiple ESDB::Identity objects, one for each AI level, and then change the identity-mapping code accordingly.

**Comment by dsjoerg** [comment]
_From @eric-hu on January 11, 2016 6:42_

Just revisiting this issue since I've had it as a to-do to investigate this, but I don't think I'll have time any time soon.  Feel free to close this issue if you'd like to.

**Comment by eric-hu** [comment]
Totally understand, I wanted to look into it over the last few months too.  Unfortunately I don't think I'll have the time either.  It looks like I don't have closing rights to this issue.  If you'd like to close it, that's all right with me.

**Comment by dsjoerg** [comment]
@eric-hu let's leave this open, now that ggtrackerstack makes it easier to mess with ggtracker, maybe someone else will be inspired by this issue to fix this problem.
